src/webview/lib/RecipeWebview.ts

```diff
@@ -30,8 +30,6 @@ class RecipeWebview {
     ipcRenderer.on('poll', () => {
       this.loopFunc();
 
-      debug('Poll event');
```

Comment:
This was probably the worst offender which made it impossible to look at logs in an unfiltered state inside the dev tools.